llama_stack/apis/inference/inference.py

```diff
@@ -913,6 +913,7 @@ class OpenAIEmbeddingData(BaseModel):
     """
 
     object: Literal["embedding"] = "embedding"
+    # TODO: consider dropping str and using openai.types.embeddings.Embedding instead of OpenAIEmbeddingData
```

Comment:
do we have a tracker for that TODO?

Reply:
it's a note for the next person who looks at fixing mypy errors.

i'm going by https://github.com/llamastack/llama-stack/blob/main/pyproject.toml#L244